paver.sh

```diff
@@ -478,6 +478,7 @@ check_os_version() {
     # supported for the current OS and the current detected version.
     # The fourth parameter is used to return through eval the relevant numbers
     # for naming the Python package for the current OS, as detailed above.
+    # Beware, this function is full of bash'isms.
```

Comment:
what is the point of this comment? the script starts with declaring that it needs bash

Reply:
Not everyone uses `bash` all the time… I thought of it as a reminder to self that the following lines make repeated use of incompatible Bash extensions to the POSIX standard. So that I don't fall for checking local manuals for some stuff. :-]

If it's too much, I'll remove it.

Reply:
we can leave it, but for me is clear that this script only works with bash... as otherwise we would just change the first line of it :)

please merge. thanks!